index.html

```diff
@@ -5,132 +5,179 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
     <title>Mobile Version Skeleton</title>
 </head>
 <body>
-    
-<header>
-    <nav class="nav-bar"> 
-        
-        <div class="button-a">
-        <a href="" class="my-logo">My Logo</a>
-        </div>
-       
-        <div class="button-b">
-            <div class="burger-button"></div>
-        </div>
 
-    </nav>
-</header>
+    <header>
+        <nav class="nav-bar"> 
+            
+            <div class="button-a">
+            <a href="" class="my-logo">My Logo</a>
+            </div>
+           
+            <div class="button-b">
+                <div class="burger-button"></div>
+            </div>
+    
+        </nav>
+    </header>
 
 <section id="body">
+
     <div class="title">
         <h1>I'm Abel <br>Glad to see <br>you!</h1>
     </div>
+    
     <div class="text">
         <p>I am a software developer! I can help you build a product,  
             Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     </div>
+    
     <div class="button-c">
         <a href="#" class="letsconnect">LET'S CONNECT</a>
```

Comment:
- [ ] The text `Let's Connect` is a heading and as a result, should be placed in an `hx` tag like an `h2` or `h3` tag for best semantic practices. Kindly rectify this issue 👍